schemas/context/profile.schema.json

```diff
@@ -86,6 +91,24 @@
           "description":
             "The unit ID within the organization owning the profile. This ID can be used to reference the organization details maintained in another dataset."
         },
+        "xdm:preferredLanguage": {
```

Comment:
there is no information about what format this takes?  BPC-47?  Other?

Reply:
Added format information (ISO-639-1 language codes) in latest commit.

Reply:
Only -1 codes?  That's very old school.  Are you sure that is what is being used?  Why arent' we using the modern BCP-47 approach to language specification?

Reply:
Added a schema for language, making it flexible for applications to specify language codes of their choice.

Reply:
This has been resolved, we are using BCP-47 now.